extras/indexing/src/main/java/mvm/rya/indexing/external/fluo/FluoPcjUpdaterConfig.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package mvm.rya.indexing.external.fluo;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import org.apache.hadoop.conf.Configuration;
+
+import com.google.common.base.Optional;
+
+import mvm.rya.indexing.accumulo.ConfigUtils;
+
+/**
+ * Configuration values required to initialize a {@link FluoPcjUpdater}.
+ */
+public final class FluoPcjUpdaterConfig {
+
+    // Defines which Fluo application is running for this instance of Rya.
+    public static final String FLUO_APP_NAME = "rya.indexing.pcj.fluo.fluoAppName";
+
+    // Values that define which Accumulo instance hosts the Fluo application's table.
+    public static final String ACCUMULO_ZOOKEEPERS = ConfigUtils.CLOUDBASE_ZOOKEEPERS;
```

Comment:
We could add a Jira Improvement ticket to do that if you would like to update them in the future.